src/npm/index.js

```diff
@@ -153,14 +222,17 @@ export async function getDownloads(pkgs) {
       },
     };
   });
-}
 
-export function getDependents(pkgs) {
-  // we return 0, waiting for https://github.com/npm/registry/issues/361
-  return Promise.all(
-    pkgs.map(() => ({
-      dependents: 0,
-      humanDependents: '0',
-    }))
-  );
+  datadog.timing('npm.getDownloads', Date.now() - start);
+  return all;
 }
+
+export default {
```

Comment:
I would prefer this to be 
```suggestion
export {
```

and `import * as npm from './npm'` to be able to tree shake it better

Reply:
oops I just remembered this isn't a library, but still prefer the form I suggest. 

This will still make it easier for us to move functions into a separate file, because we will be able to write `export { nameOfFun } from './xxx'` in this file

Reply:
While I agree on the principle, 2 things comes into my mind:
- We usually never move code like that
- It's hard to onboard a codebase where name are too generic. 
  `findAll()` can literally mean anything, with `npm.findAll()` I don't have to go up, find the file to understand the line etc... another solution can be to have the full purpose in the function name but it's really redundant when you are inside the file, e.g: `npmFindAll()`.
  For one line it's obviously not a problem but over a whole codebase it does make a difference. Just my 2 cents on this, if you still want named export I'm fine with it, don't hesitate to request it 👍 ☺️

Reply:
This is why I'm suggesting `import * as npm from './npm'`, and then doing `npm.findAll`

Reply:
indeed, that would work 👌